templates/centos-stream-10.yaml

```diff
@@ -0,0 +1,27 @@
+# minimumLimaVersion: "1.0.3"
+
+images:
+# Try to use release-yyyyMMdd image if available. Note that release-yyyyMMdd will be removed after several months.
+- location: "https://cloud.centos.org/centos/10-stream/x86_64/images/CentOS-Stream-GenericCloud-10-20241216.0.x86_64.qcow2"
+  arch: "x86_64"
+  digest: "sha256:94090f04d517b6b2d8e95f949ceb861fd5cd0a6f2076c20c906e44c6c43f566c"
+- location: "https://cloud.centos.org/centos/10-stream/aarch64/images/CentOS-Stream-GenericCloud-10-20241216.0.aarch64.qcow2"
+  arch: "aarch64"
+  digest: "sha256:1034f42dc140e5d4f517458ad584dcb03fdbefc3d5721946c79070b4fe2d796e"
+# Fallback to the latest release image.
+# Hint: run `limactl prune` to invalidate the cache
+- location: "https://cloud.centos.org/centos/10-stream/x86_64/images/CentOS-Stream-GenericCloud-10-latest.x86_64.qcow2"
+  arch: "x86_64"
+- location: "https://cloud.centos.org/centos/10-stream/aarch64/images/CentOS-Stream-GenericCloud-10-latest.aarch64.qcow2"
+  arch: "aarch64"
+mountTypesUnsupported: ["9p"]
+mounts:
+- location: "~"
+- location: "/tmp/lima"
+  writable: true
+firmware:
+  # CentOS Stream 10 still requires legacyBIOS
+  legacyBIOS: true
+cpuType:
+  # Does not seem to boot with "qemu64"
+  x86_64: "Haswell-v4"
```

Comment:
This seems to fix #3022 (Intel MacBook Pro 2020) too

Reply:
qemu64-v1 is not supported by EL10, it requires v3 (or v2)

Reply:
I thought that it would use "host" by default, on Intel?

Reply:
On ARM it uses "qemu64", and the kernel just reboots infinitely

Reply:
Right, that would be expected. So this was when testing on ARM, got it.

Reply:
Found a comparison table: https://www.qemu.org/docs/master/system/i386/cpu.html

Surprised to know that qemu64 sticks to `x86_64-v1` by default 👀

Reply:
Did you have the qemu line that was used, originally. It was `-cpu host`, right?

Reply:
Updated the PR to explain the issue for both ARM (qemu64) and Intel (host)

Reply:
> Did you have the qemu line that was used, originally. It was -cpu host, right?

It is chosen by Lima itself; "qemu64" for ARM, "host" ("-pdpe1gb") for Intel

Reply:
(GitHub posts seem to be temporarily delayed?)

Reply:
> (GitHub posts seem to be temporarily delayed?)

We can use Slack if wanting to have a conversation.